app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] Please make sure the devise is also confirmable and ask for confirmation of email as requested in the project requirements.

![image](https://github.com/diarisdiakite/Blog-app/assets/83803414/abe05b4c-d8fb-45b0-9afc-b0bee093d7b9)